### ISSUE TITLE
ganga output messages in triple batches

### ISSUE THREAD

**Issue by afortiorama** [ganga-devs/ganga#1214]
Hi,

I'm using /cvmfs/ganga.cern.ch/runGanga.sh and the already high number of messages seems now to be sent three times each. Ganga is already quite verbose could you look at why it repeats 3 times the same messages?

For example

```
aforti@lxplus108>ganga
WARNING  Ganga will soon be depending on Python 2.7. You have Python 2.6 installed. See https://github.com/ganga-devs/ganga/wiki/Python-2.7
WARNING  Ganga will soon be depending on Python 2.7. You have Python 2.6 installed. See https://github.com/ganga-devs/ganga/wiki/Python-2.7
WARNING  Ganga will soon be depending on Python 2.7. You have Python 2.6 installed. See https://github.com/ganga-devs/ganga/wiki/Python-2.7

*** Welcome to Ganga ***
Version: 6.7.4
Documentation and support: http://cern.ch/ganga
Type help() or help('index') for online help.

This is free software (GPL), and you are welcome to redistribute it
under certain conditions; type license() for details.


INFO     reading config file /afs/cern.ch/user/a/aforti/.gangarc
INFO     reading config file /afs/cern.ch/user/a/aforti/.gangarc
INFO     reading config file /afs/cern.ch/user/a/aforti/.gangarc

[...]
WARNING  some files not successfully downloaded
WARNING  some files not successfully downloaded
WARNING  some files not successfully downloaded
ERROR    fail to download job output: 6
ERROR    fail to download job output: 6
INFO     Job 6 Running PostProcessor hook
INFO     Job 6 Running PostProcessor hook
INFO     Job 6 Running PostProcessor hook
INFO     Job 6 Setting Location of LocalFile: log.txt
INFO     Job 6 Setting Location of LocalFile: log.txt
INFO     Job 6 Setting Location of LocalFile: log.txt
INFO     Job 6 Putting File LocalFile: log.txt
INFO     Job 6 Putting File LocalFile: log.txt
INFO     Job 6 Putting File LocalFile: log.txt
INFO     job 6 status changed to "completed"
INFO     job 6 status changed to "completed"
INFO     job 6 status changed to "completed"
```

**Comment by rob-c** [comment]
Glad I'm not the only one who sees this from time to time.
if you try from a clean bash session do you see the same errors? If not what type of system do you run this on and do you see the same running ganga dev?

**Comment by afortiorama** [comment]
Yes, I see them. Both on lxplus and my local UI SL6.9 and SL6.8 respectively. I haven't tried on a CentOS7 machine yet. I haven't tried ganga-dev.

**Comment by mesmith75** [comment]
Is this still an issue? I think it has been fixed in the most recent versions.

**Comment by drmarkwslater** [comment]
I've just tried this myself and it seems to have been fixed by 7.0.1 at least which is what's on CVMFS. I'll close and @afortiorama can re-open/create a new issue if she sees it again.
